src/index.html

```diff
@@ -22,6 +25,374 @@
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple.html"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung.html"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#smartphones.html"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops-&-computers.html"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#gadgets.html"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#tablets.html"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#photo.html"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#video.html"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac-img"
+        />
+
+        <div class="card__info">
+          <h2 class="card__info-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__info-product-code">Product code: 195434</p>
+        </div>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-count">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
```

Comment:
The `data-qa` attribute is used for testing purposes and should not be present in the production code unless it is specifically required for testing.